transform_utils.ml

```diff
@@ -19,10 +19,76 @@ let change_instrs (transform : pc -> instruction_change) ({formals; instrs} : an
       | Replace is ->
         acc_instr (pc+1) (List.rev_append is acc) true
       | Insert is ->
+        assert(match[@warning "-4"] instrs.(pc) with | Label _ -> false | _ -> true);
```

Comment:
Please add a comment explaining this insert. (The next time I see this code, I'll probably forget why there's an assert.)